clusterloader2/pkg/measurement/common/network-policy/manifests/dep-test-client-policy-creation.yaml

```diff
@@ -46,6 +47,8 @@ spec:
           -MaxTargets={{.MaxTargets}}
           -MetricsPort={{.MetricsPort}}
           -AllowPolicyName={{.AllowPolicyName}}
+          -np_type={{.NetworkPolicy_Type}}
```

Comment:
Let's follow proper format like the original arguments. For example, `-NetworkPolicyType` instead of `np_type`. Also, remove _ for consistency in naming, like `NetworkPolicyType` instead of `NetworkPolicy_Type`

Reply:
Does this repo have a style guide?